variables.tf

```diff
@@ -454,17 +454,30 @@ variable "webhooks" {
   # }]
 }
 
-variable "plaintext_secrets" {
-  description = "(Optional) Configuring actions secrets. For details please check: https://www.terraform.io/docs/providers/github/r/actions_secret.html"
-  type        = map(string)
+variable "secrets" {
```

Comment:
i would suggect to keep plaintext_secrets and add a second map of encrypted_secrets so we do not break the API..